pact-mode.el

```diff
@@ -44,45 +44,45 @@
   "Regexp match for valid pact identifiers.")
 
 (defconst pact-font-lock-keywords
-  (eval-when-compile
```

Comment:
`pact-identifier` can't be found during compilation if this is wrapped in `eval-when-compile`, at least in Spacemacs.